### ISSUE TITLE
APA 7th style not respecting pages notation (at least in Zotero)

### ISSUE THREAD

**Issue by kirmaha** [citation-style-language/styles#6239]
Hello,
_This is an issue report I posted on [Zotero forum](https://forums.zotero.org/discussion/99785/issue-report-apa-7th-style-not-respecting-pages-notation) first, but it didn't get much attention. As I'm not familiar with the process of CSL creation and update, I'm not sure where to report this. I hope the right place is here but if it's not, could you show me the way ? Thanks !_

I noticed that there are two errors in APA 7th style when using in-text citations : it should always show "p." for single page and "pp." for several pages, whether they are separated with "-" or with "," .

1. In English :
- It shows "pp." when I specify page numbers in "1-99" format (pages 1 to 99), which is correct.
- It shows "p." when I specify page numbers in "1,99" format (pages 1 and 99). I should show "pp."

2. In French : it always shows "p." when I enter several pages, whether they are separated with "-" or with "," .

Here are the APA references about this :
- English : https://apastyle.apa.org/style-grammar-guidelines/citations/quotations/page-numbers
- French : https://cdn.scribbr.com/wp-content/uploads/2020/09/Manuel-APA-de-Scribbr-7eme-edition.pdf (p. 27)

Thank you !

**Comment by POBrien333** [comment]
@bwiernik I guess you missed the thread in the forum while on holidays.

**Comment by bwiernik** [comment]
I'll update the APA styles tomorrow

**Comment by bwiernik** [comment]
> It shows "p." when I specify page numbers in "1,99" format (pages 1 and 99). I should show "pp."

You should put a space after the comma. Does it use pp then?



For French, I've never seen "pp" used for plural pages in French. Do you have an example of that? French adaptations of APA style generally do not use pp for multiple pages

**Comment by kirmaha** [comment]
Hello !
Thank you for your answer !

> You should put a space after the comma. Does it use pp then?
Indeed, with the English version it does. It then include that space in the in-text citation, which I never noticed ! Thank you.

> For French, I've never seen "pp" used for plural pages in French. Do you have an example of that? French speaking adaptations of APA style generally do not use pp for multiple pages
I did some guidelines research, and it looks like both are suggested. As long as there's no official French speaking version of APA's guidelines, several guidelines coexist.
1. http://profmcouture.ca/apa/normes_apa_francais.pdf (Canada, based on Belgian guidelines) suggests to use "p." for plural.
2. http://doclib.uliege.be/apa/Wiki.jsp?page=CitationsDansLeTexte (Belgian, university) suggests to use "pp." for plural.
3. https://bib.vinci.be/opac_css/doc_num_data.php?explnum_id=4208 (Belgian, college) tells us that both can be used for plural, and that they would opt for "p.".

I didn't find many examples to give you, as I don't read much in French and those texts are usually not in APA. Some "pp." are used in other styles (not necessary for in-text citations), for example : _Nadot, M. (2003). Mesure des prestations soignantes dans le système de santé. Recherche en soins infirmiers, 73, 116‑122_. But there's no way I can state that "pp." is the common use.

Of course, it appeared to me as a mistake because I learned to use "pp." and thought that it was the only way to go with APA. As there's no official French speaking version, I personality don't see what justifies to move away from the original English speaking guidelines, but that's just an opinion. Now, I don't know which one is the best on a community point of view.

Have a nice day !
